src/modules/tpm/src/lib/Tpm2Utils.h

```diff
@@ -0,0 +1,284 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <errno.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <ScopeGuard.h>
+#include <Tpm.h>
+
+#define INT_MAX 0x7FFFFFF
+#define TPM_RESPONSE_MAX_SIZE 4096
+#define TPM_COMMUNICATION_ERROR -1
+#define TPM_PATH "/dev/tpm0"
+
+static const uint8_t g_getTpmProperties[] =
+{
+    0x80, 0x01,                 // TPM_ST_NO_SESSIONS
+    0x00, 0x00, 0x00, 0x16,     // commandSize
+    0x00, 0x00, 0x01, 0x7A,     // TPM_CC_GetCapability
+    0x00, 0x00, 0x00, 0x06,     // TPM_CAP_TPM_PROPERTIES
+    0x00, 0x00, 0x01, 0x00,     // Property: TPM_PT_FAMILY_INDICATOR
+    0x00, 0x00, 0x00, 0x66      // propertyCount (102)
+};
+
+class Tpm2Utils
+{
+    public:
+        static int UnsignedInt8ToUnsignedInt64(uint8_t* inputBuf, uint32_t inputBufSize, uint32_t dataOffset, uint32_t dataLength, uint64_t* output)
```

Comment:
Do not indent declarations after public/private/protected labels.

Instead of:

class Foo
{
    public:
         Blah;

Do:

class Foo
{
public:
         Blah;